### ISSUE TITLE
gimme genome mm10 error

### ISSUE THREAD

**Issue by c-guzman** [simonvh/gimmemotifs#65]
```
$ gimme genome gimme_genomes/ mm10
Retrieving gene annotation for mm10
Using http://hgdownload.cse.ucsc.edu/goldenPath/mm10/database/knownGene.txt.gz
zcat /tmp/tmpfxuf6a7g.gz | cut -f1-11 | /home/cag104/miniconda3/envs/bioinfo/bin/genePredToBed /dev/stdin /home/cag104/miniconda3/envs/bioinfo/share/gimmemotifs/genes/mm10.bed
Downloading mm10 genome
Trying to download http://hgdownload.soe.ucsc.edu/goldenPath/mm10/bigZips/chromFa.tar.gz
Unpacking
tar: gimme_genomes/mm10: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
Creating index
Traceback (most recent call last):
  File "/home/cag104/miniconda3/envs/bioinfo/bin/gimme", line 496, in <module>
    args.func(args)
  File "/home/cag104/miniconda3/envs/bioinfo/lib/python3.6/site-packages/gimmemotifs/commands/genome.py", line 23, in genome
    get_genome(args.genomebuild, args.fastadir, args.indexdir)
  File "/home/cag104/miniconda3/envs/bioinfo/lib/python3.6/site-packages/gimmemotifs/genome_index.py", line 241, in get_genome
    g = g.create_index(genome_dir, index_dir)
  File "/home/cag104/miniconda3/envs/bioinfo/lib/python3.6/site-packages/gimmemotifs/genome_index.py", line 344, in create_index
    raise IOError(msg)
OSError: No fastafiles found in /gpfs/data01/heinzlab/home/cag104/reference_data/Mus_musculus/UCSC/mm10/gimme_genomes/mm10 with extension in .fasta,.fa,.fsa
```

**Comment by simonvh** [comment]
Could it be that the directory `gimme_genomes` does not exist? I do think `gimme genome` should create it, so I would count this as a bug anyway, but it might be a quick fix for your problem.

**Comment by simonvh** [comment]
I just released a new version of GimmeMotifs (0.12.0) that has changed the genome backend. The download of genomes is now handled by [genomepy](http://github.com/simonvh/genomepy). If you have any trouble with genomepy, feel free to open an issue there.
